TAF_EasyRest/TestFramework/Pages/ModeratorPanelPestaurantsPage.cs

```diff
@@ -37,7 +42,23 @@ public ModeratorPanelRestaurantsPage(IWebDriver driver) : base(driver)
 
         #region Methods
 
-        public ModeratorPanelRestaurantsPage ClickUnapprovedTabButton()
+        private void FillRestaurantsGrid()
+        {
+            restaurants = new List<UnapprovedRestaurantPageComponent>(СountRestaurants());
+            for (int i = 0; i < restaurants.Count; i++)
```

Comment:
I completely don't understand that logic. What is the reason for passing the same driver instance into the page component class?